src/easyscience/fitting/minimizers/minimizer_lmfit.py

```diff
@@ -18,7 +18,8 @@
 from lmfit.model import ModelResult
 
 from easyscience.Objects.new_variable import Parameter as NewParameter
-from easyscience.Objects.ObjectClasses import BaseObj
+
+# from easyscience.Objects.ObjectClasses import BaseObj ruff complained about this was unused
```

Comment:
Cool! this fixes circular dependencies I noticed earlier and means I don't have to do a separate PR.